webauthn/helpers/structs.py

```diff
@@ -19,25 +18,27 @@ class WebAuthnBaseModel(BaseModel):
     - Encodes bytes to Base64URL
     - Converts snake_case properties to camelCase
 
-    `Model.parse_raw()` (from JSON):
+    `Model.model_validate_json()` (from JSON):
     - Decodes Base64URL to bytes
     - Converts camelCase properties to snake_case
     """
 
-    class Config:
-        json_encoders = {bytes: bytes_to_base64url}
-        json_loads = json_loads_base64url_to_bytes
-        alias_generator = snake_case_to_camel_case
-        allow_population_by_field_name = True
+    model_config = ConfigDict(
+        alias_generator=snake_case_to_camel_case,
+        populate_by_name=True,
+    )
 
-    @validator("*", pre=True, allow_reuse=True)
-    def _validate_bytes_fields(cls, v, field: ModelField):
+    @field_validator("*", mode="before")
```

Comment:
It's also possible to handle base64 decoding of `bytes` fields here, but I found no way to use the same `"*"` pattern for `@field_serializer` so I wasn't able to make that work both ways. So because of that I decided to use a separate type instead.